### PR TITLE
Upgrade to v1.0.5, compatible with IDEA 15.0

### LINKED ISSUE #17: Plugin throws error in IDEA 15.0
The plugin v1.0.4 worked in IDEA 14.1.5, but not anymore in 15.0. When trying to start a run config, the event log says:

    NoSuchMethodError: com.intellij.execution.configurations.GeneralCommandLine.setEnvParams(Ljava/util/Map;)V

### PR DESCRIPTION
**Pull request by kriegaex**
This should fix #17.

### CODE REVIEW

File: src/org/intellij/lang/batch/runner/BatchRunner.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ protected RunContentDescriptor doExecute(final Project project,
         ExecutionResult executionResult = state.execute(env.getExecutor(), this);
         if (executionResult == null || executionResult.getExecutionConsole() == null) return null;
 
-        final RunContentBuilder contentBuilder = new RunContentBuilder(this, executionResult, env);
+        final RunContentBuilder contentBuilder = new RunContentBuilder(/*this,*/ executionResult, env);
```
Comment (aefimov) [inline src/org/intellij/lang/batch/runner/BatchRunner.java]: Maybe just delete?

File: resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<idea-plugin version="2">
+    <id>Batch Scripts Support</id>
+    <name>Batch Scripts Support</name>
+    <version>1.0.5</version>
+    <vendor>Alexey Efimov</vendor>
```
Comment (aefimov) [inline resources/META-INF/plugin.xml]: I guess you need to be author too.

File: src/org/intellij/lang/batch/runner/BatchRunner.java (hunk 1 above)
Comment (kriegaex) [inline src/org/intellij/lang/batch/runner/BatchRunner.java]: Yes, feel free to delete in a subsequent commit after you accept the PR. I just forgot about it.

File: resources/META-INF/plugin.xml (hunk 2 above)
Comment (kriegaex) [inline resources/META-INF/plugin.xml]: No no, too much honour for a simple quick & dirty fix. :-) I do not even understand what the plugin does exactly and have never developed a plugin. This one is your baby, just mention me in the release noted, that's enough. I am just happy I managed to fix the plugin even though I use a deprecated method.

**Pull request merged**